### DISCUSSION TITLE
Codespases not working for me

### DISCUSSION THREAD

**Discussion by Oritsejolomi** [community/community#50626] [Codespaces]
### Select Topic Area

Bug

### Body

I keep getting this message when trying to start my codespaces how can I fix it?
<img width="440" alt="Screen Shot 2023-03-21 at 6 29 23 PM" src="https://user-images.githubusercontent.com/79635828/226693055-90ea68fe-2988-4804-96d3-7195401d974a.png">

**Comment by osortega** [comment]
Hello @Oritsejolomi 👋 
I'm sorry this is happening to you, are you connecting through VS Code Desktop or Web?
Do you mind sharing your codespace name or url you are connecting to so we can take a closer look?

**Reply by Oritsejolomi** [reply]
I’m trying on both Mac and web. This is the error I get on the mac app. But on web the page just doesn’t load. My codespace name is ‘Oritsejolomi’

**Reply by osortega** [reply]
@Oritsejolomi do you mind sharing your codespace name/url so we can take a closer look?

**Comment by FabioAlves8889** [comment]
sim

Em qui., 23 de mar. de 2023 15:12, Osvaldo Ortega ***@***.***>
escreveu:
